README.md

```diff
@@ -9,6 +9,8 @@ constraints and special considerations of mobile and edge deployments.
 See [our website](https://iree-org.github.io/iree/) for project details, user
 guides, and instructions on building from source.
 
+![status_badge](https://github.com/iree-org/iree/actions/workflows/ci.yml/badge.svg?branch=main)
```

Comment:
Let's make this image link to the CI results and also narrow to just the `push` event. Without filtering by event type we also get `main` branches in forks >_> https://github.com/iree-org/iree/actions/workflows/ci.yml?query=branch%3Amain

![image](https://user-images.githubusercontent.com/4010439/184193816-bd4f9313-91d6-4314-8f99-d7eb18172103.png)

This suggestion should work:

```suggestion
[![CI Status](https://github.com/iree-org/iree/actions/workflows/ci.yml/badge.svg?query=branch%3Amain+event%3Apush)](https://github.com/iree-org/iree/actions/workflows/ci.yml?query=branch%3Amain+event%3Apush)
```